.github/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ### New features since last release
 
+* Add `mid-circuit measurements` support to `lightning.gpu`'s single-GPU backend.
```

Comment:
How come the support is not for MPI? Is it an outdated changelog entry :thinking:

Reply:
MPI dropped